mewarpx/mewarpx/assemblies.py

```diff
@@ -93,22 +89,63 @@ def __init__(self, center_x, center_z, radius, V, T, WF, name,
             T (float): Temperature (K)
             WF (float): Work function (eV)
             name (str): Assembly name
-            install_in_fieldsolver (float): If True and the Assembly is an
+            install_in_fieldsolver (bool): If True and the Assembly is an
                 embedded boundary it will be included in the WarpX fieldsolver
         """
         super(Cylinder, self).__init__(V=V, T=T, WF=WF, name=name)
-
+        # Y is always treated as the height
         self.center_x = center_x
         self.center_z = center_z
+        self.height = 1 # m
```

Comment:
As far as I can tell this is not used anywhere, specifically not in the `isinside` function or the implicit function. So the physical object actually has y = infinity. Can we therefore remove it?